Appveyor build failing again

https://ci.appveyor.com/project/snowman2/rioxarray/builds/30418759
```
pytest --cov-report term-missing --cov=rioxarray
ImportError while loading conftest 'C:\projects\rioxarray\test\conftest.py'.
test\conftest.py:5: in <module>
    from rioxarray.rioxarray import UNWANTED_RIO_ATTRS
rioxarray\__init__.py:8: in <module>
    import rioxarray.rioxarray  # noqa
rioxarray\rioxarray.py:18: in <module>
    import rasterio.warp
c:\miniconda3-x64\envs\test\lib\site-packages\rasterio\__init__.py:22: in <module>
    from rasterio._base import gdal_version
E   ImportError: DLL load failed: The specified module could not be found.

```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -4,7 +4,7 @@
 
 environment:
   matrix:
-    - PYTHON_VERSION: "3.6"
+    - PYTHON_VERSION: "3.7"
       MINICONDA: "C:\\Miniconda3-x64"
 
 install:
@@ -22,12 +22,12 @@ install:
 #-------------------------------------------------------------------------------
 # Install rioxarray
 #-------------------------------------------------------------------------------
-  - pip install -e .[all]
+  - python -m pip install -e .[all]
 
 build: false
 
 test_script:
-  - pytest --cov-report term-missing --cov=rioxarray
+  - python -m pytest --cov-report term-missing --cov=rioxarray
   - flake8 --ignore=E731,W503,W504 --exclude --max-complexity 10 --max-line-length 88 rioxarray/
   - flake8 --max-line-length 88 tests/unit/ tests/functional/ tests/integration
   - black --check .

